logsrvd/logsrvd.h

```diff
@@ -98,6 +98,7 @@ struct connection_closure {
     struct iolog_file iolog_files[IOFD_MAX];
     int iolog_dir_fd;
     int sock;
+    char ipaddr[INET6_ADDRSTRLEN];
```

Comment:
INET6_ADDRSTRLEN won't be available on non-IPv6 systems.

Reply:
fixed